src/misc/soft_limiter.cpp

```diff
@@ -176,7 +176,7 @@ void SoftLimiter::ScaleOrCopy(const std::vector<float> &in,
 {
 	assert(samples >= 2); // need at least one frame
 	auto in_start = in.begin() + channel;
-	const auto in_end = in.begin() + channel + samples;
+	const auto in_end = in.begin() + ( channel + samples - 1 );
```

Comment:
minor: no spaces around parens (just run the formatter on it)

Reply:
Thanks for catching that.